packages/dht/src/connection/WebRTC/NodeWebRtcConnection.ts

```diff
@@ -81,7 +82,7 @@ export class NodeWebRtcConnection extends EventEmitter<Events> implements IConne
 
         this.connectingTimeoutRef = setTimeout(() => {
             logger.trace('connectingTimeout, this.closed === ' + this.closed)
-            this.doClose()
+            this.doClose('OTHER')
```

Comment:
Disconnection case OTHER is set in quite many disconnect cases. What is it used to designate?

Reply:
I think we should split it to 'CONNECTION_TIMEOUT', 'INCOMING_CLOSE' etc. Left it like that for now, and will come back to it when also the downwards migration works.